staging/src/k8s.io/client-go/scale/client_test.go

```diff
@@ -274,9 +275,7 @@ func TestGetScale(t *testing.T) {
 
 	for _, groupResource := range groupResources {
 		scale, err := scaleClient.Scales("default").Get(context.TODO(), groupResource, "foo", metav1.GetOptions{})
-		if !assert.NoError(t, err, "should have been able to fetch a scale for %s", groupResource.String()) {
-			continue
-		}
+		require.NoError(t, err, "should have been able to fetch a scale for %s", groupResource.String())
```

Comment:
Same as above. If you think converting assert to require is correct, that should be a different PR because it is an actual behavior change